<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="27.326" tests="10" failures="10">
  <testsuite name="Root Suite" timestamp="2025-10-21T13:58:10" tests="0" time="0.000" failures="0">
  </testsuite>
  <testsuite name="GOP Multiple Executors E2E" timestamp="2025-10-21T13:58:10" tests="2" file="/Users/marian.irudayanathan/IdeaProjects/probate-frontend/test/end-to-end/paths/probate/multipleExecutors.js" time="5.910" failures="2">
    <testcase name="GOP Multiple Executors E2E EN - Multiple Executors Journey - Main applicant; Stage 1: Enter deceased and executor details - With Idam @e2enightly @e2enightly-pr" time="0.485" classname="EN - Multiple Executors Journey - Main applicant; Stage 1: Enter deceased and executor details - With Idam @e2enightly @e2enightly-pr">
      <failure message="TestConfigurator.getBefore: Using proxy - ERROR: 
Error stack:
AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1714:7)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)" type="Error"><![CDATA[Error: TestConfigurator.getBefore: Using proxy - ERROR: 
Error stack:
AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1714:7)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)
    at IDAMHelper.createAUser (test/end-to-end/helpers/IDAMHelper.js:41:23)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Test.<anonymous> (test/end-to-end/paths/probate/multipleExecutors.js:31:9)]]></failure>
    </testcase>
    <testcase name="GOP Multiple Executors E2E CY - Multiple Executors Journey - Main applicant; Stage 1: Enter deceased and executor details - With Idam @e2enightly @e2enightly-pr" time="0.490" classname="CY - Multiple Executors Journey - Main applicant; Stage 1: Enter deceased and executor details - With Idam @e2enightly @e2enightly-pr">
      <failure message="TestConfigurator.getBefore: Using proxy - ERROR: 
Error stack:
AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1714:7)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)" type="Error"><![CDATA[Error: TestConfigurator.getBefore: Using proxy - ERROR: 
Error stack:
AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1714:7)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)
    at IDAMHelper.createAUser (test/end-to-end/helpers/IDAMHelper.js:41:23)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Test.<anonymous> (test/end-to-end/paths/probate/multipleExecutors.js:31:9)]]></failure>
    </testcase>
  </testsuite>
  <testsuite name="GOP Multiple Executors E2E - EE Yes Journey" timestamp="2025-10-21T13:58:16" tests="2" file="/Users/marian.irudayanathan/IdeaProjects/probate-frontend/test/end-to-end/paths/probate/multipleExecutors.EE.yes.js" time="5.348" failures="2">
    <testcase name="GOP Multiple Executors E2E - EE Yes Journey EN - Multiple Executors Journey - Main applicant EE yes journey - With Idam @e2enightly @e2enightly-pr" time="0.487" classname="EN - Multiple Executors Journey - Main applicant EE yes journey - With Idam @e2enightly @e2enightly-pr">
      <failure message="TestConfigurator.getBefore: Using proxy - ERROR: 
Error stack:
AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1714:7)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)" type="Error"><![CDATA[Error: TestConfigurator.getBefore: Using proxy - ERROR: 
Error stack:
AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1714:7)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)
    at IDAMHelper.createAUser (test/end-to-end/helpers/IDAMHelper.js:41:23)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Test.<anonymous> (test/end-to-end/paths/probate/multipleExecutors.EE.yes.js:31:9)]]></failure>
    </testcase>
    <testcase name="GOP Multiple Executors E2E - EE Yes Journey CY - Multiple Executors Journey - Main applicant EE yes journey - With Idam @e2enightly @e2enightly-pr" time="0.484" classname="CY - Multiple Executors Journey - Main applicant EE yes journey - With Idam @e2enightly @e2enightly-pr">
      <failure message="TestConfigurator.getBefore: Using proxy - ERROR: 
Error stack:
AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1714:7)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)" type="Error"><![CDATA[Error: TestConfigurator.getBefore: Using proxy - ERROR: 
Error stack:
AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1714:7)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)
    at IDAMHelper.createAUser (test/end-to-end/helpers/IDAMHelper.js:41:23)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Test.<anonymous> (test/end-to-end/paths/probate/multipleExecutors.EE.yes.js:31:9)]]></failure>
    </testcase>
  </testsuite>
  <testsuite name="GOP Multiple Executors E2E - EE No Journey" timestamp="2025-10-21T13:58:22" tests="2" file="/Users/marian.irudayanathan/IdeaProjects/probate-frontend/test/end-to-end/paths/probate/multipleExecutors.EE.no.js" time="5.365" failures="2">
    <testcase name="GOP Multiple Executors E2E - EE No Journey EN - Multiple Executors Journey - Main applicant EE no journey - With Idam @e2enightly @e2enightly-pr" time="0.489" classname="EN - Multiple Executors Journey - Main applicant EE no journey - With Idam @e2enightly @e2enightly-pr">
      <failure message="TestConfigurator.getBefore: Using proxy - ERROR: 
Error stack:
AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1714:7)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)" type="Error"><![CDATA[Error: TestConfigurator.getBefore: Using proxy - ERROR: 
Error stack:
AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1714:7)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)
    at IDAMHelper.createAUser (test/end-to-end/helpers/IDAMHelper.js:41:23)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Test.<anonymous> (test/end-to-end/paths/probate/multipleExecutors.EE.no.js:30:9)]]></failure>
    </testcase>
    <testcase name="GOP Multiple Executors E2E - EE No Journey CY - Multiple Executors Journey - Main applicant EE no journey - With Idam @e2enightly @e2enightly-pr" time="0.487" classname="CY - Multiple Executors Journey - Main applicant EE no journey - With Idam @e2enightly @e2enightly-pr">
      <failure message="TestConfigurator.getBefore: Using proxy - ERROR: 
Error stack:
AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1714:7)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)" type="Error"><![CDATA[Error: TestConfigurator.getBefore: Using proxy - ERROR: 
Error stack:
AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1714:7)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)
    at IDAMHelper.createAUser (test/end-to-end/helpers/IDAMHelper.js:41:23)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Test.<anonymous> (test/end-to-end/paths/probate/multipleExecutors.EE.no.js:30:9)]]></failure>
    </testcase>
  </testsuite>
  <testsuite name="GOP-Single Executor" timestamp="2025-10-21T13:58:27" tests="2" file="/Users/marian.irudayanathan/IdeaProjects/probate-frontend/test/end-to-end/paths/probate/gopSingleExecutor.en.test.js" time="5.350" failures="2">
    <testcase name="GOP-Single Executor EN -GOP Single Executor E2E  - With Idam @e2enightly @e2enightly-pr" time="0.486" classname="EN -GOP Single Executor E2E  - With Idam @e2enightly @e2enightly-pr">
      <failure message="TestConfigurator.getBefore: Using proxy - ERROR: 
Error stack:
AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1714:7)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)" type="Error"><![CDATA[Error: TestConfigurator.getBefore: Using proxy - ERROR: 
Error stack:
AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1714:7)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)
    at IDAMHelper.createAUser (test/end-to-end/helpers/IDAMHelper.js:41:23)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Test.<anonymous> (test/end-to-end/paths/probate/gopSingleExecutor.en.test.js:29:9)]]></failure>
    </testcase>
    <testcase name="GOP-Single Executor CY -GOP Single Executor E2E  - With Idam @e2enightly @e2enightly-pr" time="0.488" classname="CY -GOP Single Executor E2E  - With Idam @e2enightly @e2enightly-pr">
      <failure message="TestConfigurator.getBefore: Using proxy - ERROR: 
Error stack:
AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1714:7)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)" type="Error"><![CDATA[Error: TestConfigurator.getBefore: Using proxy - ERROR: 
Error stack:
AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1714:7)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)
    at IDAMHelper.createAUser (test/end-to-end/helpers/IDAMHelper.js:41:23)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Test.<anonymous> (test/end-to-end/paths/probate/gopSingleExecutor.en.test.js:29:9)]]></failure>
    </testcase>
  </testsuite>
  <testsuite name="GOP-Single Executor - EE Yes Journey" timestamp="2025-10-21T13:58:32" tests="2" file="/Users/marian.irudayanathan/IdeaProjects/probate-frontend/test/end-to-end/paths/probate/gopSingleExecutor.en.test.EE.yes.js" time="5.351" failures="2">
    <testcase name="GOP-Single Executor - EE Yes Journey EN -GOP EE yes journey  - With Idam @e2enightly @e2enightly-pr" time="0.483" classname="EN -GOP EE yes journey  - With Idam @e2enightly @e2enightly-pr">
      <failure message="TestConfigurator.getBefore: Using proxy - ERROR: 
Error stack:
AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1714:7)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)" type="Error"><![CDATA[Error: TestConfigurator.getBefore: Using proxy - ERROR: 
Error stack:
AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1714:7)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)
    at IDAMHelper.createAUser (test/end-to-end/helpers/IDAMHelper.js:41:23)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Test.<anonymous> (test/end-to-end/paths/probate/gopSingleExecutor.en.test.EE.yes.js:30:9)]]></failure>
    </testcase>
    <testcase name="GOP-Single Executor - EE Yes Journey CY -GOP EE yes journey  - With Idam @e2enightly @e2enightly-pr" time="0.489" classname="CY -GOP EE yes journey  - With Idam @e2enightly @e2enightly-pr">
      <failure message="TestConfigurator.getBefore: Using proxy - ERROR: 
Error stack:
AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1714:7)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)" type="Error"><![CDATA[Error: TestConfigurator.getBefore: Using proxy - ERROR: 
Error stack:
AggregateError [ECONNREFUSED]: 
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1714:7)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)
    at IDAMHelper.createAUser (test/end-to-end/helpers/IDAMHelper.js:41:23)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Test.<anonymous> (test/end-to-end/paths/probate/gopSingleExecutor.en.test.EE.yes.js:30:9)]]></failure>
    </testcase>
  </testsuite>
</testsuites>